frontend/src/p5/p5components/Circle.js

```diff
@@ -12,7 +12,9 @@ class Circle {
 
     draw(){
         // Draw the circle
-        this.p.fill(this.color);
+        this.p.fill(220);
```

Comment:
Fine for now, but color needs to be set in the sketch files themselves if we want there to be user customizability